menus/darwin.cson

```diff
@@ -165,6 +164,7 @@
         submenu: [
           { label: 'Open In Dev Mode…', command: 'application:open-dev' }
           { label: 'Run Package Specs', command: 'window:run-package-specs' }
+          { label: 'Reload Window', command: 'window:reload' }
```

Comment:
My theory on this was essentially:

* Open in Dev Mode might be targeted by people as being the first in the list
* Toggle Dev Tools might be targeted by people as being the last in the list (and it is targeted that way by me)
* Run Package Specs feels more related to Open in Dev Mode to me

But yeah, now that you mention it ... alphabetic order makes a little more sense.